src/components/TableNew/TableHeaderNode.tsx

```diff
@@ -0,0 +1,9 @@
+import { cn } from '@/lib/utils'
+import { FC } from 'react'
+import { CommonComponentProps } from '../commonProps'
+
+export interface TableHeaderNodeProps extends CommonComponentProps {}
+
+export const TableHeaderNode: FC<TableHeaderNodeProps> = ({ children, className }) => {
```

Comment:
I still does not understand the purpose of this component. 

I can understand that is the children from the `TableHeaderCell` but what is the purpose of wrapping the content with it?

To have the components in a column while the `TableHeaderCell` in a row?

Reply:
Yes, basically. It's just a layout component in a way.